velox/functions/sparksql/Arithmetic.h

```diff
@@ -318,6 +318,19 @@ struct IsNanFunction {
   }
 };
 
+template <typename T>
```

Comment:
nit: T -> TExec, TInput -> T for readability

Reply:
Upated, thanks.